todo_app/app.py

```diff
@@ -1,26 +1,99 @@
+import os
 from flask import Flask, redirect, url_for,  render_template
 from todo_app.data import session_items
 from todo_app.flask_config import Config
 from flask import request
+import requests
 
 app = Flask(__name__)
-app.config.from_object(Config)
+app.config.from_object(Config)   
+
+KEY=os.getenv("KEY")
+TOKEN=os.getenv("TOKEN")
+    
+
+class TransportAPI:
+    def __init__(self):
+        api_keys = get_application_keys()
+        self.payload = api_keys
+
```

Comment:
This looks like code from the workshop - we should remove that here

Reply:
Removed